### PR TITLE
better message on column not found

### LINKED ISSUE #379: BUG-Inputting the wrong column name in a chart reports empty data
**Description**
Inputting the wrong column name in a chart visual element will report that the dataset used is empty when it is not. This leads the user to think that the issue is with the dataset instead of the column name.

**How to reproduce**
Running this code where I made a mistake on the y column of the chart visual element:

```python
from taipy import Gui
import pandas as pd

data = pd.DataFrame({"a": [1, 2, 3], "b": [4, 5, 6]})

page = """
<|{data}|chart|type=bar|x=a|y=hello|>
"""

Gui(page).run()
```

Leads to this error:
```
--- 1 warning(s) were found for page '/' in variable 'page' ---
 - Warning 1: Error column "hello" is not present in the Dataframe "Empty DataFrame
Columns: [a, b]
Index: []".
```

**Expected behavior**
The error should not mention that the DataFrame is empty, only that the column is not found

**Runtime environment**
 - OS: Windows 11
 - Browser: Chrome

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**
based on [srinivaspavan9](https://github.com/srinivaspavan9)'s work
linter compliance

resolves #379

### CODE REVIEW

File: taipy/gui/_renderers/utils.py [hunk 1]
```diff
@@ -31,14 +33,30 @@ def _get_columns_dict_from_list(
...
+                    else list(value.keys())
+                    if isinstance(value, (dict, _MapDict))
+                    else value
+                    if isinstance(value, (list, tuple))
+                    else []
+                )
+
+            if cols and (col not in cols):
+                _warn(
+                    f'Error column "{col}" is not present. Available columns: {cols}.'  # noqa: E501
```
Comment (FabienLelaquais) [inline taipy/gui/_renderers/utils.py]:
```suggestion
                    f'Column "{col}" is not present. Available columns: {cols}.'  # noqa: E501
```

File: taipy/gui/_renderers/utils.py [hunk 2]
```diff
@@ -31,14 +33,30 @@ def _get_columns_dict_from_list(
...
+                )
+            else:
+                _warn("The 'value' argument is of an unsupported type. Expected DataFrame, dict, list, or tuple.")
```
Comment (FabienLelaquais) [inline taipy/gui/_renderers/utils.py]:
```suggestion
                _warn("The 'data' property value is of an unsupported type. Only DataFrame, dict, list, or tuple are supported.")
```
But is this enough?

**Review by FabienLelaquais — DISMISSED** [review]
Make sure what the path to that error message is...

**Review by FabienLelaquais — DISMISSED** [review]
👌

**Review by FabienLelaquais — DISMISSED** [review]
👍

**Review by FabienLelaquais — APPROVED** [review]
👌👌👌

**Review by dinhlongviolin1 — APPROVED** [review]

**Review by trgiangdo — APPROVED** [review]

**Pull request merged**